docs/en/start/secret/start.md

```diff
@@ -0,0 +1,44 @@
+# use Secret API to obtain secret
+## What is Secret API
+The secret API is used to obtain secret from file, env, k8s, etc
+
+Get all API and secret support
+## Quick start
+
+This example shows how to obtain the secret in file, env and k8s through the Layotto secret API
+
+
+
+### Step 1:  Run Layotto
+
+After downloading the project code to the local, switch the code directory and compile:
+
+```shell
+cd ${projectpath}/cmd/layotto
+go build
```

Comment:
I suggest we wrap this line in a separate script block and add an annotation to improve performance:
~~~suggestion
```shell @if.not.exist layotto
# build it
go build -o layotto
```
~~~